packages/app-mobile/components/ScreenHeader/index.tsx

```diff
@@ -643,7 +654,14 @@ class ScreenHeaderComponent extends PureComponent<ScreenHeaderProps, ScreenHeade
 
 		const menuComp =
 			!menuOptionComponents.length || !showContextMenuButton ? null : (
-				<Menu onSelect={value => this.menu_select(value)} style={this.styles().contextMenu}>
+				<Menu onSelect={value => this.menu_select(value)} style={this.styles().contextMenu} onOpen={() => {
+					setTimeout(() => {
+						const menuFirstItem = findNodeHandle(this.menuOptionsRef.current);
+						if (menuFirstItem) {
+							AccessibilityInfo.setAccessibilityFocus(menuFirstItem);
+						}
+					}, 500);
```

Comment:
I tried moving menuComp to a separate component then using `useEffect` and a few other things but it did not work out, so resorted to using `setTimeout` in the end